glide-core/src/protobuf/redis_request.proto

```diff
@@ -135,6 +135,7 @@ enum RequestType {
     Rename = 91;
     DBSize = 92;
     Brpop = 93;
+    Blpop = 94;
```

Comment:
These numbers will have to change based on merges, keep an eye on it when rebasing from main.

Reply:
got it